packages/react/test/specs/utils/__snapshots__/felaRenderer-test.tsx.snap

```diff
@@ -79,40 +44,12 @@ exports[`felaRenderer basic styles are rendered 1`] = `
 `;
 
 exports[`felaRenderer keyframe colors are rendered 1`] = `
-"@-webkit-keyframes k1 {
```

Comment:
This looks broken, I expect that generated keyframes should stay there

Reply:
Thanks for the catch! I'll take a look

Reply:
I've spent a good bit of time trying to figure this out, as far as I can tell it works, at least it does in all the examples that I've changed it is. But there is a breakdown in the fella-react-jest framework which is causing classnames to not get passed down. I'm wondering if React.Clone element isn't working/getting called in the Animation component. I'll take another look tomorrow.

Reply:
`Animation` component depends on theme styles which I think is a bug.
Created an issue #2247 and pushed workaround for the snapshot test to this PR.